web-client/src/ustc-ui/Pagination/Paginator.tsx

```diff
@@ -1,51 +1,60 @@
+import { Button } from '@web-client/ustc-ui/Button/Button';
 import React from 'react';
-import ReactPaginate from 'react-paginate';
 
 export const Paginator = ({
-  breakClassName,
-  forcePage,
-  id,
-  marginPagesDisplayed = 3,
+  currentPageIndex,
   onPageChange,
-  pageCount,
-  pageRangeDisplayed,
+  totalPages,
 }: {
-  breakClassName?: string;
-  marginPagesDisplayed?: number;
-  pageCount: number;
-  id?: string;
-  pageRangeDisplayed: number;
-  onPageChange: (selectedItem: { selected: number }) => void;
-  forcePage: number;
+  totalPages: number;
+  currentPageIndex: number;
+  onPageChange: (currentPage) => void;
 }) => {
+  let currentPage = currentPageIndex + 1;
+
   return (
-    <>
-      <nav
-        aria-label="Pagination"
-        className="usa-pagination margin-bottom-0 margin-top-0"
-        id={id}
-      >
-        <ReactPaginate
-          activeLinkClassName="paginator-current"
-          breakClassName={`usa-pagination__item ${breakClassName}`}
-          breakLinkClassName="usa-pagination__button cursor-pointer border-none"
-          className="usa-pagination__list"
-          disabledClassName="hide"
-          forcePage={forcePage}
-          marginPagesDisplayed={marginPagesDisplayed}
-          nextClassName="usa-pagination__item"
-          nextLinkClassName="usa-pagination__link usa-pagination__next-page cursor-pointer"
-          pageClassName="usa-pagination__item usa-pagination__page-no"
-          pageCount={pageCount}
-          pageLinkClassName="usa-pagination__button cursor-pointer"
-          pageRangeDisplayed={pageRangeDisplayed}
-          previousClassName="usa-pagination__item"
-          previousLinkClassName="usa-pagination__link usa-pagination__previous-page cursor-pointer"
-          renderOnZeroPageCount={null}
-          onPageChange={onPageChange}
-        />
-      </nav>
-    </>
+    <nav
+      aria-label="Pagination"
+      className="usa-pagination margin-bottom-0 margin-top-0"
+      role="navigation"
+    >
+      <ul className="usa-pagination__list">
+        {currentPage > 1 && (
+          <li className="usa-pagination__item">
+            <Button
+              link
+              aria-label="Previous"
+              className="usa-pagination__link usa-pagination__previous-page cursor-pointer"
+              disabled={currentPage === 1}
```

Comment:
Do we need this if we are only rendering it for `currentPage > 1`? Similar question for the next button below.

Reply:
This is a good idea, fixed! I'd love to hear more about this accessibility recommendation, since I think we are violating it dozens of times throughout the app